scripts/fetch_dependencies.sh

```diff
@@ -25,50 +25,127 @@ do
   esac
 done
 
+XC_FOLDER=XCIntegration
+
 function updatePods {
-  echo "> Grabbing SDKs using CocoaPods integration."
```

Comment:
Why are you removing the echos from its own function? I don't see any benefit on having different logs for when we pass an integration parameter or when we don't

Reply:
It's a safety measure. Depending on the version of bash used, echo can act as a `return` from a local func, e.g. when you want to return a value from a func you would just echo it.